build_kernel.sh

```diff
@@ -10,62 +11,92 @@ OUTPUT_DIR=$DIR/output
 BOOT_IMG=./boot.img
 cd $DIR
 
+if [[ "$(uname)" == 'Darwin' ]]; then
+  echo "---------------   macOS support   ---------------"
+  echo "Ensure you are in a Case-sensitive APFS volume to build the AGNOS kernel."
+  echo "https://github.com/commaai/agnos-builder?tab=readme-ov-file#development---macos"
+  echo "-------------------------------------------------"
+  echo "Press any key to continue or CTRL+C to abort..."
+  read -n 1 -s
```

Comment:
can we detect instead of showing a prompt on every build?

Reply:
resolved!